integration/integration.test.js

```diff
@@ -0,0 +1,166 @@
+'use strict'
+
+const Lab = require('lab')
+const Code = require('code')
+
+const Plugin = require('..')
+const Seneca = require('seneca')
+
+const lab = exports.lab = Lab.script()
+const describe = lab.describe
+const it = lab.it
+const expect = Code.expect
+
+
+function pick_value (obj) {
+  return obj.value
+}
+
+
+describe('Seneca Zipkin Tracer', function () {
+  let client
+  let server
+  let annotations
+
+  function fakeTransport (data) {
+    annotations.push.apply(annotations, data.annotations.map(pick_value))
+  }
+
+  function test (done) {
+    client.ready(function () {
+      client.act('local:test', function (err, res) {
+        try {
+          expect(err).to.not.exist()
+          expect(res).to.equal({hello: 'world'})
+          expect(annotations).to.equal([ 'cs', 'cs', 'sr', 'ss', 'cr', 'cr' ])
```

Comment:
Yeah, may be a good idea.
In short, we track each seneca "action" as spans in the trace, even when the action is totally local and doesn't call a server. 
So in this case we should track `cs` and `cr` for `local:test` and `cs`,`sr`,`ss`,`cr` for `remote:test`